packages/dropdowns/src/styled/multiselect/StyledMultiselectInput.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { StyledInput } from '../select/StyledInput';
+
+const COMPONENT_ID = 'dropdowns.multiselect_input';
+
+interface IStyledMultiselectInputProps extends ThemeProps<DefaultTheme> {
```

Comment:
This should pass type-checking, but it may be cleaner to follow the syntax we use in other components.

`ThemeProps<DefaultTheme>` is added through the `styled-components` types. Rather than add them to the styled element props it may be more accurate to include it as `& ThemeProps<DefaultTheme>` to your styling helpers.